### PR TITLE
GitHub fetcher sql base

### PR DESCRIPTION
**Pull request by apelisse**
Create SQL Connection for the fetcher.

### CODE REVIEW

File: github-stats/fetcher/mysql.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+package main
+
+import (
+	"fmt"
+
+	"github.com/jinzhu/gorm"
+	"github.com/spf13/cobra"
+)
+
+// MySQLConfig is a specific to this database
```
Comment (spxtr) [inline github-stats/fetcher/mysql.go]: nit: grammar

File: github-stats/fetcher/mysql.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+type MySQLConfig struct {
+	Host     string
+	Port     int
+	Db       string
```
Comment (spxtr) [inline github-stats/fetcher/mysql.go]: DB

**Comment by spxtr** [comment]
Actually, can we move the vendor/Godeps stuff so that it's under github-stats?

**Comment by apelisse** [comment]
I feel like I'm going back and forth with the vendor path. But I think it makes sense to me. I'll do that. Thanks. and fix the nits.

**Comment by apelisse** [comment]
@spxtr Moved vendor to `github-stats/`. I agree it's better. And fixed the grammar.

**Comment by spxtr** [comment]
OK, I think it's reasonable. I still don't really know how the whole system is going to work, though.

**Pull request merged**